components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -1739,28 +1473,29 @@ public void testOfficialScriptDeleteNoSudo(boolean isPrivileged, String groupPer
             Assert.assertEquals(currentScript, actualScript);
             Assert.assertFalse(isExpectSuccessDeleteOfficialScript);
         } catch (Ice.LocalException | ServerError se) {
-            /* can catch only ServerError once RawFileStoreTest.testBadFileId is fixed */
+            /* Have to catch both types of exceptions because of
+             * RawFileStoreTest.testBadFileId behavior.*/
```

Comment:
I am sorry, I wasn't thinking: you are quite right. Do feel free to leave as it was if you prefer.